src/NHibernate/Proxy/NHibernateProxyBuilder.cs

```diff
@@ -398,6 +398,7 @@ private static void ImplementGetLazyInitializer(TypeBuilder typeBuilder, MethodI
 			if (method.DeclaringType.IsInterface &&
 			    method.DeclaringType.IsAssignableFrom(parentType))
 			{
+				Delegate
```

Comment:
Hum, maybe your keyboard focus was not where you were expecting it to be when you typed this.

Reply:
Yes